styles.css

```diff
@@ -1,6 +1,11 @@
-header a {
-  text-decoration: none;
+body {
+  background-image: url(../crispy-images/pexel-photo.jpeg);
```

Comment:
if you take out the `../` does this stop working? I didn't look at the file structure

Reply:
I won't know until I merge the page and test.